sig-cli/annual-report-2024.md

```diff
@@ -4,22 +4,22 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+- Custom profiling support in kubectl debug is promoted to GA
+- Interactive delete in kubectl delete is promoted to GA
+- Transition from SPDY to Websockets is promoted to beta
+- kuberc alpha phase implementation is started and reviewed extensively
+- Removing kustomize from kubectl is discussed and rejected
+- Krew has 286 plugins
+- Kustomize new releases (latest one v5.6.0 as of now) 
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+- Kui project seems to be not actively maintained
```

Comment:
Last update was on May 17, 2023. 
Do we consider archiving this repo or ping the owners to know if they plan to develop on the repo in 2025?

Reply:
I tried to contact on Slack but no response so far. I'll add this into the next sig-cli biweekly agenda to discuss.